theano/misc/pycuda_example.py

```diff
@@ -349,8 +346,8 @@ def make_thunk(self, node, storage_map, _, _2):
         def thunk():
             z = outputs[0]
             if (z[0] is None or
-                z[0].shape != inputs[0][0].shape or
-                not z[0].is_c_contiguous()):
+                    z[0].shape != inputs[0][0].shape or
+                    not z[0].is_c_contiguous()):
                 z[0] = theano.sandbox.cuda.CudaNdarray.zeros(
                     inputs[0][0].shape)
             if inputs[0][0].shape != inputs[1][0].shape:
```

Comment:
Should that function end with `z[0] = out`?
Flake8 complains that `out` is unused, and it may be right...